backend-api/errors/CustomAPIError.js

```
@@ -0,0 +1,7 @@
+class CustomAPIError extends Error {
+  constructor(message) {
+    super(message);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)